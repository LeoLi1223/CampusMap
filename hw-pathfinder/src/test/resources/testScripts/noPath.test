# tests that no path from start node to end node is found
# and that "no path found" can be printed

CreateGraph graph1

AddNode graph1 A
AddNode graph1 B
AddNode graph1 C

AddEdge graph1 A B 5.0

FindPath graph1 A C