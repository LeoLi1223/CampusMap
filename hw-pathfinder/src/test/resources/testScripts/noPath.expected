# tests that no path from start node to end node is found
# and that "no path found" can be printed

created graph graph1

added node A to graph1
added node B to graph1
added node C to graph1

added edge 5.000 from A to B in graph1

path from A to C:
no path found